lib/diagnostic.js

```diff
@@ -41,7 +77,13 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.house.deleteone(
```

Comment:
This will attempt to delete an entire document (it's `deleteOne`, not `deleteone`).  We want to remove the `motto` key from the document that has `"House Arryn"` as the value of its `name` key.  Here is a solution using `update()` with `$unset`.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```